scripts/install-worker.sh

```diff
@@ -113,13 +116,13 @@ sudo systemctl restart sshd.service
 ### iptables ###################################################################
 ################################################################################
 sudo mkdir -p /etc/eks
-sudo mv $TEMPLATE_DIR/iptables-restore.service /etc/eks/iptables-restore.service
+sudo mv "${TEMPLATE_DIR}/iptables-restore.service" /etc/eks/iptables-restore.service
```

Comment:
`TEMPLATE_DIR` is customizable variable, defined at LINE-10.
If input path contains `space` it would became a problem.